src/backend/MemoryManager.cpp

```diff
@@ -258,7 +257,7 @@ void MemoryManager::printInfo(const char *msg, const int device)
             unit = "MB";
         }
 
-        std::cout << " |  " << std::right << std::setw(14) << kv.first << " "
+        std::cout << "|  " << std::right << std::setw(14) << kv.first << " "
```

Comment:
I know this is not part of this PR but you should look into changing this into printfs. `cout` should only be used in templates.